tests/orchestrator.js

```diff
@@ -0,0 +1,20 @@
+import retry from "async-retry";
+
+async function waitForAllServices() {
+  await waitForWebServer();
+
+  async function waitForWebServer() {
+    return retry(fetchStatusPage, {
+      retries: 100,
+    });
+
+    async function fetchStatusPage() {
+      const response = await fetch("http://localhost:3000/api/v1/status");
+      const responseBody = await response.json();
+    }
+  }
+}
+
```

Comment:
```suggestion
async function waitForAllServices() {
  await waitForWebServer();
}

async function waitForWebServer() {
  return retry(fetchStatusPage, {
    retries: 100,
    maxTimeout: 1000,
  })
}

async function fetchStatusPage() {
  const response = await fetch("http://localhost:3000/api/v1/status");
  await response.json();
}

```

Sugestão para redução do alinhamento de funções.